src/intervals/conversion.jl

```diff
@@ -31,16 +25,12 @@ function convert{T<:AbstractFloat, S<:Real}(::Type{Interval{T}}, x::S)
 end
 
 function convert{T<:AbstractFloat}(::Type{Interval{T}}, x::Float64)
-    II = convert(Interval{T}, rationalize(x))
-    # This prevents that rationalize(x) returns a zero when x is very small
-    if x != zero(x) && II == zero(Interval{T})
-        return Interval(parse(T, string(x), RoundDown),
-                        parse(T, string(x), RoundUp))
-    end
-    II
+    isinf(x) && return Interval{T}(x)
```

Comment:
If x is infinite, then it should be an error to convert it to an Interval (though this can happen in the constructor itself).